docs/man_pages/project/configuration/prop-print-androidversioncode.md

```diff
@@ -0,0 +1,31 @@
+prop print AndroidVersionCode
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder prop print AndroidVersionCode [--valid-value]`
+
+Prints information about the configuration of the AndroidVersionCode property.
+
+<% if((isConsole && (isNativeScript || isCordova)) || isHtml) { %>
+### Options
+* `--valid-value` - When set, prints a description for the AndroidVersionCode property.
```

Comment:
I've written that it provides a description because based on our json schemas this is the output of the command:
```Bash
$ appbuilder prop print AndroidVersionCode --valid-value
     Android version code (same as android:versionCode)
```

I know it says that it provides valid values on the main $ prop print command but for some properties that might actually not be entirely true